src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,164 @@
+
+## Performance Metrics
+
+Some aggregate functions can be applied to transaction events to help you better understand the performance characteristics of your applications.
+
+### Duration Percentiles
+
+You can aggregate transaction durations using the following aggregate functions:
+- average
+- 75%, 95%, and 99% duration percentiles
+
+One use-case for using these statistics is to help you identify transactions that are slower than your organization's target SLAs.
+
+Transaction events are sampled, so the percentiles we present within Sentry only represents the data we have. Because of sampling, data ranges, filters, or low volume of transactions, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what you may expect to see.
+
+This inability to be usefully accurate will happen more often in some columns than others. We can calculate an average with less data than a 95th percentile, but it’ll also vary by row. An index page will always get more traffic than `/settings/country/belgium/tax`.
+
+### Percentage Query
+
+This column shows the percentage of the entire query that this row represents, so you filter your results with the query and then see how much of the result set a particular transaction represents.
+
+### Requests Per Minute (RPM)
+
+Requests Per Minute is a way to measure throughput. It is the average of all requests bucketed by the minute for the current time window and query string.
+
+## Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system. It may represent the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
+
+When you initialize and load a Sentry SDK that supports tracing, it will automatically instrument parts of your running application, such as function calls and network operations. Instrumenting a database call could entail these three steps or subroutines:
+
+1. Calling a function to compose a database query
+2. Sending a query to a database
+3. Receiving and transforming results
+
+Each of these subroutines takes some time to perform (no matter how fast they could run). Sentry records these subroutines as a unit of work in the form of **spans**. Sentry can also attach useful information to spans, such as tags, additional contextual data, and a status to indicate if the subroutine failed or not.
+
+Spans can have descendant spans (or child spans). In our earlier example, the three subroutines can be spans within a larger encompassing span (the database call).
+
+Any instrumented span is part of a trace (identified by its trace id, `trace_id`), and each span has its own marker called the `span_id`, a fixed length of alphanumeric characters.
+
+An example of a span that describes a call to an external service:
```

Comment:
@untitaker do we have a better code example using `createJiraTask`? If so, will you provide the code example?

Reply:
I have no idea where createJiraTask comes from. @markstory or @dashed may know

Reply:
I think @markstory came up with this. 

Would it be better to come up with another example, or recycle an example to keep this section consistent with the rest of the tracing docs?

Reply:
I'd try to keep it consistent with the other examples. If it's better communication to create a unique example, I'm okay with that option for those reasons.

Reply:
@MimiDumpling 

Use this code sample:

```json

// updateJiraTask
{
    "trace_id": "a8233eb845154a2cae4712b28375d08d",
    "parent_span_id": "ba99c37bfb30f860",
    "span_id": "92ce7ea47d5b8a45",
    "start_timestamp": 1583868931.940406,
    "same_process_as_parent": true,
    "description": "updateJiraTask",
    "tags": { "status": "ok" },
    "timestamp": 1583868932.060615,
    "op": "jira",
    "data": {}
}

// sql.update - a child span of updateJiraTask
{
    "trace_id": "a8233eb845154a2cae4712b28375d08d",
    "parent_span_id": "92ce7ea47d5b8a45",
    "span_id": "a88e33dab5b745b9",
    "start_timestamp": 1583868931.953386,
    "same_process_as_parent": true,
    "description": "sql.update - SELECT * FROM jira_table",
    "tags": { "status": "ok" },
    "timestamp": 1583868932.03692,
    "op": "db",
    "data": {}
}
```